notifications_api_common/management/commands/register_kanalen.py

```diff
@@ -26,7 +50,15 @@ def create_kanaal(kanaal: str) -> None:
     # look up the exchange in the registry
     _kanaal = next(k for k in KANAAL_REGISTRY if k.label == kanaal)
 
-    kanalen = client.get("kanaal", params={"naam": kanaal})
+    response_data = []
```

Comment:
this variable is assigned as empty dict at line 72. I think one of the lines can be removed. 
Also why to support `list` type? I thought a dict could be enough?

Reply:
The call to this endpoints returns a list of kanalen (also the reason why the query parameter is used):
```
[
    {

        "url": "http://example.com",
        "naam": "string",
        "documentatieLink": "http://example.com",
        "filters": [
            "string"
        ]
    }
]

```